### PR TITLE
RedisImpl add

### PR DESCRIPTION
**Pull request by wp19991**
add RedisImpl

### CODE REVIEW

**Review by wp19991 — COMMENTED** [review]
redis impl

**Comment by jovany-wang** [comment]
Nice work! Will review this this night.

File: api/pom.xml [hunk 1]
```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jovany-wang) [inline api/pom.xml:1]: Why remove these files?

File: README.md [hunk 2]
```diff
@@ -4,3 +4,40 @@ A service to provide many useful and out-of-the-box components with adapting man
...
+
+## Maven
+        <dependency>
+            <groupId>org.pine</groupId>
+            <artifactId>pine-api</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
+
+
+## Basic use (Redis and Distkv)
```
Comment (jovany-wang) [inline README.md]:
```suggestion
## Basic usage (Redis and Distkv) 
```

**Comment by jovany-wang** [comment]
Also you should remove the `.class` files.

File: api/pom.xml (hunk 1 above)
Comment (wp19991) [reply api/pom.xml:1]: I think this project should have few dependencies. The original project has too many dependencies.
I did some refactoring.
In the root directory pom.xml It's about the dependencies you need.
Without the test dependencies, now you just need the following dependencies.
 
```base
       <dependency>
            <groupId>redis.clients</groupId>
            <artifactId>jedis</artifactId>
            <version>3.1.0</version>
        </dependency>
        <dependency>
            <groupId>com.distkv</groupId>
            <artifactId>distkv-client</artifactId>
            <version>0.1.4-SNAPSHOT</version>
        </dependency>
```

**Comment by jovany-wang** [comment]
Duplicate with #10

**Pull request closed without merging**